jstest/builder/lumpy.py

```diff
@@ -328,50 +328,54 @@ def dump_section_table(sections):
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="LuMPy - Linker Map Parser")
-    parser.add_argument("-o", "--output",
+    # Ignore invalid names here, because pylint sees use_stdout and args
```

Comment:
Could you please organize these steps into a separated main function and call from here? Afterwards, these variables could be remain local variables.